test/framework/parallelbuild.py

```diff
@@ -196,7 +211,7 @@ def test_build_easyconfigs_in_parallel_pbs_python(self):
     def test_build_easyconfigs_in_parallel_gc3pie(self):
         """Test build_easyconfigs_in_parallel(), using GC3Pie with local config as backend for --job."""
         try:
-            import gc3libs
+            import gc3libs  # noqa (ignore unused import)
         except ImportError:
             print "GC3Pie not available, skipping test"
             return
```

Comment:
Is GC3Pie version verified in here? If not, should it be done?

Reply:
That's done in the `gc3pie.py` implementation.

Reply:
okidoki.